src/mappings/DAOFactory/mapping.ts

```diff
@@ -99,7 +99,11 @@ export function handleProxyCreated(event: ProxyCreated): void {
 
   if (equalStrings(event.params._contractName, 'Avatar')) {
     let avatar = Avatar.bind(event.params._proxy);
-    addContract(avatar.vault(), 'Vault', version);
+    let vault = avatar.vault();
+    addContract(vault, 'Vault', version);
+    let dao = getDAO(event.params._proxy.toHex());
```

Comment:
`getDAO` will create new empty dao if it not exist.  which is not what we want.
the code should add vault only to already exist `Dao` entity